CWE/events/trieste.txt

```diff
@@ -129,207 +129,486 @@ country_event = {
 	
 }
 
```

Comment:
Is there a catch all clean up event to remove TRB?

Reply:
Will add it soon

Reply:
Thanks!

Reply:
Please also modify the Treaty of Osimo event to annex TRE and TRB.

Reply:
It does already, both are inherited and claims removed

Reply:
Okay that’s good but I think your changes haven’t been pushed to this PR yet

Reply:
I didn't indeed, now I have